Framework/Crystal/src/PredictPeaks.cpp

```diff
@@ -489,10 +492,58 @@ void PredictPeaks::calculateQAndAddToOutput(const V3D &hkl,
   if (m_sfCalculator) {
     p.setIntensity(m_sfCalculator->getFSquared(hkl));
   }
-
+  int edge = this->getProperty("EdgePixels");
```

Comment:
Ah, sorry, bad choice of line for this comment - I mean the entire check & early return.